### ISSUE TITLE
Use Pkg.test for test task

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/julia-vscode#573]
The [current](https://github.com/JuliaEditorSupport/julia-vscode/blob/84b3cd4d1e017e31ae72f3939952ede8c43c1790/src/tasks.ts#L49) behavior is essentially a bug.

The ideal solution here would be to start a julia instance, make sure that the currently active julia environment in VS Code is also active in that julia instance, and then run ``Pkg.test(...)`` on the currently open folder.

But, I don't think that works right now: as far as I can tell one cannot pass as path to ``Pkg.test(filepath)``, only the name of a package that is part of the current environment... 

@KristofferC Is there a way to call ``Pkg.test(filepath)`` with just a path, not a package name? And then have that run the ``test\runtests.jl`` file in that path, and still create the temporary environment etc?

**Comment by ZacLN** [comment]
Yes this should be fixed, don't think it's essential for 0.11.0

**Comment by KristofferC** [comment]
> Is there a way to call Pkg.test(filepath) with just a path, not a package name? And then have that run the test\runtests.jl file in that path, and still create the temporary environment etc?

No, currently only package dependencies can be tested using the Pkg framework.

**Comment by KristofferC** [comment]
An application is (in my view) just a package that also has a checked in `Manifest` and is supposed to be used by `activate`-ing the project in the application.

It has a name and such an application is tested using `Pkg.test("ApplicationName")`.
